honeycombio/resource_column.go

```diff
@@ -119,6 +133,7 @@ func resourceColumnUpdate(ctx context.Context, d *schema.ResourceData, meta inte
 
 func readColumn(d *schema.ResourceData) *honeycombio.Column {
 	return &honeycombio.Column{
+		ID:          d.Id(),
```

Comment:
I am wondering why this wasn't set already, was updates not working then? 🤔 Anyway, this is a good addition!

Reply:
Correct! Updates were a little broke 😅

Reply:
Well I guess I didn't properly test that 😅